common/src/main/kotlin/io/goooler/demoapp/common/util/ImageLoader.kt

```diff
@@ -101,12 +104,14 @@ object ImageLoader {
     @IntRange(from = 0) cornerRadius: Int,
     useCache: Boolean,
     builder: ImageRequest.Builder.() -> Unit = {}
-  ): Disposable = loadAny(data) {
-    placeholder(placeholderDrawable)
-    error(errorDrawable)
-    if (cornerRadius > 0) transformations(RoundedCornersTransformation(cornerRadius.toFloat()))
-    loadWithCache(useCache)
-    builder()
+  ) {
+    loadAny(data) {
+      placeholder(placeholderDrawable)
+      error(errorDrawable)
+      if (cornerRadius > 0) transformations(RoundedCornersTransformation(cornerRadius.toFloat()))
+      loadWithCache(useCache)
+      builder()
+    }
```

Comment:
Just impl Coil in common module now, can't access `Disposable` in other submodules.

https://github.com/Goooler/DemoApp/blob/a2870106c68979c67091a6ff6872e8170a627f2d/common/build.gradle.kts#L45